crates/transaction-pool/src/pool/blob.rs

```diff
@@ -81,11 +81,33 @@ impl<T: PoolTransaction> BlobTransactions<T> {
     }
 
     /// Returns all transactions that satisfy the given basefee and blob_fee.
-    pub(crate) const fn satisfy_attributes(
+    pub(crate) fn satisfy_attributes(
         &self,
         _best_transactions_attributes: BestTransactionsAttributes,
     ) -> Vec<Arc<ValidPoolTransaction<T>>> {
-        Vec::new()
+        let mut transactions = Vec::new();
+        {
+            let mut iter = self.by_id.iter().peekable();
+
+            while let Some((id, tx)) = iter.next() {
+                if tx.transaction.max_fee_per_blob_gas().unwrap_or_default() <
+                    _best_transactions_attributes.blob_fee.unwrap_or_default() as u128 ||
+                    tx.transaction.max_fee_per_gas() <
+                        _best_transactions_attributes.basefee as u128
+                {
```

Comment:
hmmm, since this should hold only blob transactions, we might want to exit early if `blob_fee` is `None`, instead of using `blob_fee.unwrap_or_default` here. We do this a lot in `blob.rs` but it might be worth changing overall

Reply:
ok!
I've added a check on blob_fee.is_some()! Hope I got it right